pkgs/development/python-modules/pydeck/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipykernel
+, ipywidgets
+, jinja2
+, numpy
+, pandas
+, pytestCheckHook
+, pytest
+, traitlets
+}:
+
+buildPythonPackage rec {
+    pname = "pydeck";
+    version = "0.7.1";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0bmx5q1mpdp2rx89qp1bd8b6s8a5l6zn5jyp4xny243mkz4h2xlh";
+    };
+
+    propagatedBuildInputs =  [
+      ipykernel
+      ipywidgets
+      jinja2
+      numpy
+      traitlets
+    ];
+
+    checkInputs =  [
+      pytestCheckHook
+      pytest
+      pandas
+    ];
+
+    disabledTests = [
+      # touches network
+      "test_nbconvert"
+    ];
```

`pythonImportsCheck` is an additional safe-guard and cheap.

```suggestion
    ];

  pythonImportsCheck = [
    "pydeck"
  ];

```

---

Interesting, didn't know, thanks!